tests/functional/dump-db.sh

```diff
@@ -8,19 +8,17 @@ needLocalStore "--dump-db requires a local store"
 
 clearStore
 
-path=$(nix-build dependencies.nix -o $TEST_ROOT/result)
```

Comment:
I guess it needs this
> nix-functional-tests> error: path '/nix/var/nix/builds/nix-29092-3485356921/nix-test/main/dump-db/result' is not in the Nix store

Reply:
Looks like it needs the `nix-build` invocation (confirmed via @Ericson2314 ) and not the variable itself.